src/mono/wasm/runtime/blazor/_Integration.ts

```diff
@@ -20,6 +21,27 @@ export async function loadBootConfig(config: MonoConfigInternal,) {
     Module.config = newConfig;
 }
 
+let configFiles: ConfigFile[] = [];
+
+export async function loadConfigFiles(resourceLoader: WebAssemblyResourceLoader | null, config: MonoConfigInternal) {
+    if (!resourceLoader) {
+        return;
+    }
+
+    configFiles = await WebAssemblyConfigLoader.initAsync(resourceLoader.bootConfig, config.applicationEnvironment!, config.startupOptions ?? {});
+}
+
+export function installConfigFilesToVfs() {
+    for (let i = 0; i < configFiles.length; i++) {
+        const configFile = configFiles[i];
+        Module.FS_createDataFile(
+            "/", // TODO: Maybe working directory?
```

Comment:
Can we have a different root path for the application?

Reply:
nobody asked for it, let's not do it.